analysis/stat_tests.py

```diff
@@ -179,3 +186,52 @@ def friedman_posthoc_tests(experiment_pivot_df):
     posthoc_tests['conover'] = sp.posthoc_conover_friedman(experiment_pivot_df)
     posthoc_tests['nemenyi'] = sp.posthoc_nemenyi_friedman(experiment_pivot_df)
     return posthoc_tests
+
+
+VarghaDelaneyResult = namedtuple("VarghaDelanyResult", "a12")
+
+
+def a12(measurements_x, measurements_y, alternative=None):
+    """Returns Vargha-Delaney A12 measure effect size for two distributions
+
+    A. Vargha and H. D. Delaney.
+    A critique and improvement of the CL common language effect size statistics
+    of McGraw and Wong.
+    Journal of Educational and Behavioral Statistics, 25(2):101-132, 2000
+
+    The Vargha and Delaney A12 statistic is a non-parametric effect size
+    measure.
+
+    The formula to compute A has been transformed to minimize accuracy errors
+    See: http://mtorchiano.wordpress.com/2014/05/19/effect-size-of-r-precision/
+
+    Significant levels from original paper:
+      Large   is > 0.714
+      Mediumm is > 0.638
+      Small   is > 0.556
+
+    Given observations of a metric (edges_covered or bugs_covered) for
+    fuzzer 1 (F2) and fuzzer 2 (F2), the A12 measures the probability that
+    running fuzzer 1 will yield a higher metric than running fuzzer 2."""
+
+    x_array = np.asarray(measurements_x)
+    y_array = np.asarray(measurements_y)
+    x_size, y_size = x_array.size, y_array.size
+    ranked = ss.rankdata(np.concatenate((x_array, y_array)))
+    rank_x = ranked[0:x_size]  # get the x-ranks
+
+    # Compute the A12 measure
+    rank_x_sum = rank_x.sum()
+    # A = (R1/n1 - (n1+1)/2)/n2 # formula (14) in Vargha and Delaney, 2000
+    a12_measure = (2 * rank_x_sum - x_size * (x_size + 1)) / (
+        2 * y_size * x_size)  # equivalent formula to avoid accuracy errors
+    return VarghaDelaneyResult(a12=a12_measure)
```

Comment:
yes, the `mannwhitneyu` function returns a named tuple

Reply:
I removed the `p_values` local at the end of the function and just return the new dataframe directly.  We'd need to change all the functions that call this function (Mann-Whitney and Wilcox tests) in order to remove the `statistic` argument.  Pylint is happy now, so I'm calling it a win.

Reply:
For the unused `alternative` argument, I added a `del alternative` which was suggested on the pylint page as an option.  The `unused_alternative` doesn't work because the `test_pair` function calls the statistical_test function with a named argument of `alternative`.